pyfar/constants/constants.py

```diff
@@ -548,3 +549,162 @@ def fractional_octave_filter_tolerance(
     frequencies = exact_center_frequency * G**relative_frequencies
 
     return lower_tolerance, upper_tolerance, frequencies
+
+def fractional_octave_frequencies_nominal(num_fractions:Literal[1,3]=1,
+                        frequency_range:tuple[float, float]=(20, 20e3)):
+    """Return the nominal center frequencies for octave-band and
+    one-third-octave-band filters.
+
+    Nominal center frequencies, as specified in the IEC 61260-1:2014 standard
+    [#]_ (Section 5.5 and Annex E), are standardized values that approximate
+    the exact center frequencies. They are defined from 10 Hz to 20 kHz.
+
+    Parameters
+    ----------
+    num_fractions : {1, 3}
+        The number of octave fractions. ``1`` returns octave center
+        frequencies, ``3`` returns third-octave center frequencies.
+        The default is ``1``.
+    frequency_range : array, tuple
+        The lower and upper frequency limits, the default is
+        ``(20, 20e3)`` following IEC 61260-1 [#]_.
+        E.g. ``(10, 20e3)`` would follow IEC 61672-1.
+
+    Returns
+    -------
+    nominal : numpy.ndarray of float
+        The nominal center frequencies.
+
+    References
+    ----------
+    .. [#] International Electrotechnical Commission, "IEC 61260-1:2014 -
+        Electroacoustics - Octave-band and fractional-octave-band filters -
+        Part 1: Specifications", IEC, 2014.
+
+    .. [#] International Electrotechnical Commission,
+        "IEC 61672-1:2013 - Electroacoustics - Sound level meters - Part 1:
+        Specifications", IEC, 2013.
```

Comment:
i think they need to be switched in order otherwise the references in the doc are not swapped

Reply:
@h-chmeruk @ahms5 I think this has been resolved. IEC 61260 is referenced and listed first. Please reopen, I am wrong.